forc-test/src/lib.rs

```diff
@@ -838,6 +839,10 @@ mod tests {
     const TEST_DATA_FOLDER_NAME: &str = "test_data";
     /// Name of the library package in the "CARGO_MANIFEST_DIR/TEST_DATA_FOLDER_NAME".
     const TEST_LIBRARY_PACKAGE_NAME: &str = "test_library";
+    /// Name of the contract package in the "CARGO_MANIFEST_DIR/TEST_DATA_FOLDER_NAME".
+    const TEST_CONTRACT_PACKAGE_NAME: &str = "test_contract";
+    /// Name of the predicate package in the "CARGO_MANIFEST_DIR/TEST_DATA_FOLDER_NAME".
+    const TEST_PREDICATE_PACKAGE_NAME: &str = "test_predicate";
```

Comment:
Should we add one for scripts as well?

Reply:
Ahh you are right, I totally forget about it 👍